aips/aip-124.md

```diff
@@ -0,0 +1,199 @@
+---
+aip:
+title: x-chain DAA authentication using Sign Personal Message with Sui
+author: 0xmaayan, igor-aptos
+Status: Draft
+type: Framework
+discussions-to (*optional):
+created: 06/11/2025
+---
+
+# AIP-X - x-chain DAA authentication using Sign Personal Message with Sui
+
+## Summary
+
+Implements authentication function giving Sui users native Aptos accounts,
+and allowing them to sign transaction on Aptos blockchain, directly from their Sui wallet.
+This is done using derivable account abstraction (AIP-113).
+
+## High-level Overview
+
+Allow Sui accounts (Sui Ed25519 key pairs) to create Aptos transactions,
+and be able to autheneticate and execute them directly on the Aptos blockchain, without interaction with Sui Blockchain.
+We do so by utilizing ["Sign Personal Message"](https://docs.sui.io/standards/wallet-standard#implementing-features) specification, and defining
+a message format that gives enough details to users so they understand what they are signing reasonably well.
+
+## Impact
+
+This will allow users with Sui Wallets to be able to interact with Aptos blockchain
+
+## Alternative Solutions
+
+Alternatively, wallets from other chains can implement integration with Aptos natively, providing more seamless experience to the users.
+
+## Specification and Implementation Details
+
+dApp is required to issue signPersoanlMessage request to the wallet with:
```

Comment:
`signPersoanlMessage` --> `signPersonalMessage`

Reply:
Also, it is unclear what this `signPersonalMessage` is and what it means for the "dApp [to be] required to issue signPersonalMessage".

Adding references here would help.